cmake/Options.cmake

```diff
@@ -19,3 +19,8 @@ option(GENNY_INSTALL_SSL
        "Install the SSL libraries local to the genny install."
        OFF
 )
+
+option(GENNY_BUILD_FLAT
+       "Use FHS-like build directories."
```

Comment:
somewhere let's link to FHS (I had to look it up) http://www.pathname.com/fhs/pub/fhs-2.3.html